create-container.sh

```diff
@@ -106,6 +106,16 @@ if [ ! -v BASE_PATH ] ; then
   BASE_PATH="/opt"
 fi
 
+# Test if known_hosts file exists
+if [ ! -f ~/.ssh/know_hosts ] ; then
+  touch ~/.ssh/know_hosts
+fi
+
+# Test if public key created
+if [ ! -f ~/.ssh/id_rsa.pub ] ; then
```

Comment:
Use the SSH_KEY_PATH var in this condition.

Reply:
I liked this way!

Reply:
@jsarola this change disappered, right?